### ISSUE TITLE
Add more descriptive code samples

### ISSUE THREAD

**Issue by hjrb** [fgilde/MudBlazor.Extensions#64]
### Feature request type

Other

### Is your feature request related to a problem?

Your code sample use the component properties grid which makes it very easy to try them. That is really great. But it is very hard to understand what to do it something does not work as expected and there is there code setting properties. After all I will not embed you components with the component grid but by setting properties in Razor/C# code. 

### Describe the solution you'd like

Your code sample use the component properties grid which makes it very easy to try them. That is really great. But it is hard to understand what to do if something does not work as expected and there is code setting properties. After all I will not embed your components with the component grid but by setting properties in Razor/C# code. A nice approach would be to generate sample code for the component using the input to the component properties editor. That would be a KILLER feature!

Note: the component grid should be built-in into Visual Studio. In the god old days that was a key selling point for VB, Delphi and VS. 

### Have you seen this feature anywhere else?

A nice approach would be to generate sample code for the component using the input to the component properties editor. That would be a KILLER feature!
Else provide manually crafter code samples

### Describe alternatives you've considered

A nice approach would be to generate sample code for the component using the input to the component properties editor. That would be a KILLER feature!
Else provide manually crafter code samples

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fgilde** [comment]
Not 100% sure if you need something different, but its already possible. Just change settings in der ComponentPropertyGrid and click the generate markup button and you will see the markup for your used settings.


![pg1](https://github.com/fgilde/MudBlazor.Extensions/assets/11070717/b54b1324-fded-4f29-bbb1-f12a46e78b37)
![pg2](https://github.com/fgilde/MudBlazor.Extensions/assets/11070717/6330fc46-fe67-4f6d-8695-611505ff6a98)

Or click on the run in try.mudex button to edit the code for your current settings directly online

![pg3](https://github.com/fgilde/MudBlazor.Extensions/assets/11070717/3e72fc83-96b9-4255-8d9b-1028788de711)

![trymudex](https://github.com/fgilde/MudBlazor.Extensions/assets/11070717/071446d9-e697-4144-87f6-fb203aa46a67)

**Comment by fgilde** [comment]
please let me know if you mean or want more than this

**Comment by hjrb** [comment]
> please let me know if you mean or want more than this

That is sufficient i guess. I had the issue that the animation of the gradient text did not work. But I resolved it by using the explicit CSS loading instructions. Thanks for the great library!! And as I already wrote: that component configuration with the code generation (which I learned know) is genius! Make a PR to include it into VS or create a VS Code extension 💪😊😍
